sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/internal/BaseAuthorizationTokenProvider.java

```diff
@@ -221,8 +211,8 @@ public String generateKeyAuthorizationSignature(String verb, URI uri, Map<String
                 headers);
     }
 
-    public String generateKeyAuthorizationSignatureNew(String verb, String resourceIdValue, String resourceType,
-            Map<String, String> headers) {
+    private String generateKeyAuthorizationSignatureNew(String verb, String resourceIdValue, String resourceType,
```

Comment:
naming: 
```suggestion
    private String generateNewKeyAuthorizationSignature(String verb, String resourceIdValue, String resourceType,
```

Reply:
@kushagraThapar New here is not relevant. just drop the "New" in the method name.

Reply:
@moderakh , I just changed the scope of the method, not renamed it in this PR. Will leave it as it is for now. Small implementation detail can be changed later.